client/signup/step-wrapper/index.jsx

```diff
@@ -65,26 +74,62 @@ export default React.createClass( {
 		}
 	},
 
+	recordComparePlansClick() {
+		analytics.ga.recordEvent( 'Upgrades', 'Clicked Compare Plans Link' );
+	},
+
+	renderComparePlans() {
+		const { stepName } = this.props;
+		if ( stepName !== 'plans' || ! isSkipPlansTestEnabled() ) {
+			return null;
+		}
+
+		const selectedSite = sites.getSelectedSite();
+		const url = plansLink( '/start/plans/compare', selectedSite );
+		const compareString = this.translate( 'Compare Plan Features' );
+
+		return (
+			<a href={ url } className="step-wrapper__compare-plans" onClick={ this.recordComparePlansClick }>
+				<Gridicon icon="clipboard" size={ 18 } />
+				{ compareString }
```

Comment:
I think there's enough logic here to be useful to extract from the main render. The different buttons pop into and out of existence depending on your step so it's easier to consider them on their own. But consider yourself vindicated.